packages/ods/src/components/range/src/components/ods-range/ods-range.scss

```diff
@@ -44,4 +45,22 @@
     width: 16px;
     height: 16px;
   }
+
+  &__ticks {
+    position: absolute;
+    inset: 0 calc(range.$ods-input-range-size-thumb / 2);
+    z-index: range.$range-z-index-dual - 2;
+
+    &__tick {
+      position: absolute;
+      border-radius: 6px;
+      background-color: range.$range-background-color;
+      width: 3px;
+      height: 18px;
```

Comment:
should be the same as the thumb, (declare a var that you'll use on the thumb style also)